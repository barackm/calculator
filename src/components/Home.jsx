import React from 'react';

const Home = () => {
  const name = 'ddd';
  return <div>{name}</div>;
};

export default Home;
