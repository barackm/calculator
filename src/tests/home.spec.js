import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import Home from '../components/Home';

describe('Home Component', () => {
  it('Should match spanshot', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <Home />
        </BrowserRouter>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
