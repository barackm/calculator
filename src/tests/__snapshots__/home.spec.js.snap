// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Component Should match spanshot 1`] = `
<header
  className="masthead"
>
  <div
    className="container px-4 px-lg-5 h-100"
  >
    <div
      className="row gx-4 gx-lg-5 h-100 align-items-center justify-content-center text-center"
    >
      <div
        className="col-lg-8 align-self-end"
      >
        <h1
          className="text-white font-weight-bold"
        >
          Welcome to our page!
        </h1>
        <hr
          className="divider"
        />
      </div>
      <div
        className="col-lg-8 align-self-baseline"
      >
        <p
          className="text-white-75 mb-5 hero-text"
        >
          Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex 
        </p>
        <a
          className="btn-home"
          href="/calculator"
          onClick={[Function]}
        >
          Find Out More
        </a>
      </div>
    </div>
  </div>
</header>
`;
